sdk/keyvault/azure-security-keyvault-keys/src/json_web_key.cpp

```diff
@@ -1,28 +1,128 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // SPDX-License-Identifier: MIT
 
+#include <azure/core/internal/json/json_optional.hpp>
 #include <azure/keyvault/common/internal/base64url.hpp>
 
 #include "azure/keyvault/keys/details/key_constants.hpp"
+#include "azure/keyvault/keys/details/key_serializers.hpp"
 #include "azure/keyvault/keys/json_web_key.hpp"
+#include "azure/keyvault/keys/key_curve_name.hpp"
 
+#include <map>
 #include <string>
+#include <vector>
 
 using namespace Azure::Security::KeyVault::Keys;
 using namespace Azure::Core::Json::_internal;
 using namespace Azure::Security::KeyVault::_internal;
 
-void Azure::Security::KeyVault::Keys::to_json(
-    Azure::Core::Json::_internal::json& j,
-    JsonWebKey const& p)
+namespace {
+void ParseStringOperationsToKeyOperations(
+    std::vector<KeyOperation>& keyOperations,
```

Comment:
The keyOperations is the out param,  Most of your APIs have it as the last, this one has it as the first.  I'd make the output last for consistency